orbs/shared/jobs/save_cache.yaml

```diff
@@ -7,6 +7,9 @@ parameters:
     default: _json_key
     description: Username to use when fetching images from a registry
     type: string
+  pre_test_script:
+    description: Environment variable that contains a shell path to run before running tests in CircleCI
```

Comment:
This needs to be removed - right? You didn't change the save cache script from what I can tell ...

Reply:
This is needed for the `save_cache` job because the test object is gatting passed in on this [line](https://github.com/getoutreach/devbase/blob/main/.circleci/config.yml#L80)

Reply:
Huh. I clearly need to read more docs here to understand what's going on. :)

Can you update this description to match the other one?

Reply:
Huh. I clearly need to read more docs here to understand what's going on. :)

Can you update this description to match the other one?

If the tests are being invoked, do you also need an `environment` block that sets the environment variable?

Reply:
Yes. There is an env block that is set in the test resource defined [here](https://github.com/getoutreach/devbase/pull/520/files#diff-b493fedbc0e0080b57980c59870e378f2a69d198e9f6b3ccdd4a638dc05f5e03R47)